source/config.js

```diff
@@ -1,16 +1,20 @@
 module.exports = function (bot) {
-    var welcomeFmt = 'Welcome to the JavaScript chat! Please review the {0}. ' +
-        'Please don\'t ask if you can ask or if anyone\'s around; just ask ' +
-        'your question, and if anyone\'s free and interested they\'ll help.';
+    var welcomeFmt = "Welcome to the JavaScript chat! Please review the {rulesLink}. " +
```

Comment:
FWIW I think that we can use template strings :D

Reply:
I also thought so, but I decided to stick with supplant for the sake of consistency with the rest of the codebase.